Arcs/Arcs.cpp

```diff
@@ -207,18 +290,247 @@ double calcSequenceIdentity(const std::string& line, const std::string& cigar, c
 
     return si;
 }
+/* Returns true if the contig sequence contains ATGC or IUPAC codes */
+static inline bool checkContigSequence(std::string seq) {
+	for (int i = 0; i < static_cast<int>(seq.length()); i++) {
+		char c = toupper(seq[i]);
+		if (c != 'A' && c != 'T' && c != 'G' && c != 'C' && c != 'N' && c != 'M'
+				&& c != 'R' && c != 'W' && c != 'S' && c != 'Y' && c != 'K'
+				&& c != 'V' && c != 'H' && c != 'D' && c != 'B') {
+			std::cout << c << std::endl;
+			return false;
+		}
+	}
+
+	return true;
+}
+
+/* Returns true if files in vector are in same format.
+ * If they are all in same format pass-by-reference parameter allAlignment=1 tells they are all in alignment format 
+ * and allAlignment=0 if they are all in read file format.*/
+bool checkSameFormat(std::vector<std::string> fileNames, bool &allAlignment){
```

Comment:
Will the program print an error message and exit if all the files are NOT in the same format?

Reply:
Yes.